pkg/resource/v1/certconfig/delete.go

```diff
@@ -42,3 +42,29 @@ func (r *Resource) newDeleteChange(ctx context.Context, obj, currentState, desir
 
 	return currentCertConfigs, nil
 }
+
+func (r *Resource) newDeleteChangeForUpdatePatch(ctx context.Context, obj, currentState, desiredState interface{}) ([]*v1alpha1.CertConfig, error) {
+	currentCertConfigs, err := toCertConfigs(currentState)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+	desiredCertConfigs, err := toCertConfigs(desiredState)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "debug", "finding out which certconfigs have to be deleted")
```

Comment:
FYI we started to use a new structured log format in operatorkit. Would be cool to use this in new logs. See https://github.com/giantswarm/operatorkit/blob/39f1a4387b0e8ffb148f54ca8b15eee7af537ca6/framework/framework.go#L555. The point is the `level` and `message` key. I think this will turn out valuable with the new EFK stacks.